server/neptune/workflows/internal/deploy/workflow.go

```diff
@@ -76,6 +76,9 @@ func newRunner(ctx workflow.Context, request Request, terraformWorkflow func(ctx
 			InstallationToken: request.Repository.Credentials.InstallationToken,
 		},
 	}
+
+	// TODO: We should actually probably pass this with the revision because a revision
```

Comment:
Could be a silly question 😅 but does our implementation handle root changes while the workflow is still running since we setup the root only when we instantiate the runner?

Reply:
nope, that's why i added this TODO so that we can support it.